src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -5,70 +5,425 @@
 using Microsoft.ML.Runtime.Internal.CpuMath.Core;
 using System;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     [BestFriend]
     internal static partial class CpuMathUtils
     {
         // The count of bytes in Vector128<T>, corresponding to _cbAlign in AlignedArray
-        private const int Vector128Alignment = 16;
+        private const int CbAlign = 16;
 
         [MethodImplAttribute(MethodImplOptions.AggressiveInlining)]
         public static int GetVectorAlignment()
-            => Vector128Alignment;
+            => CbAlign;
 
-        public static void MatrixTimesSource(bool transpose, AlignedArray matrix, AlignedArray source, AlignedArray destination, int stride) => SseUtils.MatTimesSrc(transpose, matrix, source, destination, stride);
+        private static bool Compat(AlignedArray a)
+        {
+            Contracts.AssertValue(a);
```

Comment:
Moving code like this is really hard to review.  Did you make any changes to this code, or is it a straight copy from Sse.cs to this file?

Reply:
it is straight copy from sse.cs

Reply:
The only addition is the function GetVectorAlignment